Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -439,7 +439,9 @@ public class ClusterSystem: DistributedActorSystem, @unchecked Sendable {
             self.cluster.down(member: myselfMember)
         }
 
-        self.settings.plugins.stopAll(self)
+        Task {
+            await self.settings.plugins.stopAll(self)
+        }
```

Comment:
Hmmm we may need to wait for the plugins to stop before we continue shutdown -- so shutdown must become async. We can't spin off a new task I guess as that means concurrent stoppping and shutting down

Reply:
`stop` is now blocking and non-throwing.

Reply:
Oh I see, then never threw before in reality even hm..

That'll work for now, thanks! May need to become async, we'll see...